solr/server/solr/configsets/sample_techproducts_configs/conf/managed-schema

```diff
@@ -195,39 +195,35 @@
-   <dynamicField name="*_t"  type="text_general"    indexed="true"  stored="true"/>
-   <dynamicField name="*_txt" type="text_general"   indexed="true"  stored="true" multiValued="true"/>
-   <dynamicField name="*_en"  type="text_en"    indexed="true"  stored="true" multiValued="true"/>
-   <dynamicField name="*_b"  type="boolean" indexed="true" stored="true"/>
-   <dynamicField name="*_bs" type="boolean" indexed="true" stored="true"  multiValued="true"/>
-   <dynamicField name="*_f"  type="pfloat"  indexed="true"  stored="true"/>
-   <dynamicField name="*_fs" type="pfloat"  indexed="true"  stored="true"  multiValued="true"/>
-   <dynamicField name="*_d"  type="pdouble" indexed="true"  stored="true"/>
-   <dynamicField name="*_ds" type="pdouble" indexed="true"  stored="true"  multiValued="true"/>
-
+      for fields via the specification of patterns to match field names.
+      EXAMPLE:  name="*_i" will match any field ending in _i (like myid_i, z_i)
+      RESTRICTION: the glob-like pattern in the name attribute must have a "*" only at the start or the end.  -->
+
+   <dynamicField name="*_i"   type="pint"     indexed="true"  stored="true"/>
+   <dynamicField name="*_is"  type="pints"    indexed="true"  stored="true"/>
+   <dynamicField name="*_s"   type="string"   indexed="true"  stored="true" />
+   <dynamicField name="*_ss"  type="strings"  indexed="true"  stored="true"/>
+   <dynamicField name="*_l"   type="plong"    indexed="true"  stored="true"/>
+   <dynamicField name="*_ls"  type="plongs"   indexed="true"  stored="true"/>
+   <dynamicField name="*_b"   type="boolean"  indexed="true"  stored="true"/>
+   <dynamicField name="*_bs"  type="booleans" indexed="true"  stored="true"/>
+   <dynamicField name="*_f"   type="pfloat"   indexed="true"  stored="true"/>
+   <dynamicField name="*_fs"  type="pfloats"  indexed="true"  stored="true"/>
+   <dynamicField name="*_d"   type="pdouble"  indexed="true"  stored="true"/>
+   <dynamicField name="*_ds"  type="pdoubles" indexed="true"  stored="true"/>
    <dynamicField name="*_dt"  type="pdate"    indexed="true"  stored="true"/>
-   <dynamicField name="*_dts" type="pdate"    indexed="true"  stored="true" multiValued="true"/>
-   <dynamicField name="*_p"  type="location" indexed="true" stored="true"/>
+   <dynamicField name="*_dts" type="pdates"   indexed="true"  stored="true"/>
+   <dynamicField name="*_t"   type="text_general" indexed="true" stored="true" multiValued="false"/>
+   <dynamicField name="*_txt" type="text_general" indexed="true" stored="true"/>
 
-   <dynamicField name="*_c"   type="currency" indexed="true"  stored="true"/>
+   <dynamicField name="random_*" type="random"/>
+   <dynamicField name="ignored_*" type="ignored"/>
 
-   <dynamicField name="ignored_*" type="ignored" multiValued="true"/>
-   <dynamicField name="attr_*" type="text_general" indexed="true" stored="true" multiValued="true"/>
```

Comment:
Is this removal of `attr_*` intentional? I see it's still in the `_default` config set. Will it be removed there as well?

Such changes trip up integration tests for a client library that run against Solr 7, 8 and 9 with the same data. I'm trying to determine what's the future-proof course of action here.

Reply:
thanks @thomascorthals for asking!     I want to move us in the direction of having a couple of configsets that we ship with Solr that are actually meant to be used by end users...   Like `_default`.   Right now, `sample_techproducts_configs` is a bit of an odd duck, we use it to demo new features in Solr, we use it in our unit tests, and of course, a lot of folks base their first search schema on it!   So it's always changing.

I'm also interested in having some configsets that are used by the schema designer that you can import and then use in your own schemas...   There is a PR out there for that...

However, back to your original question, I would recommend `_default` for backwards compat testing.   Or maybe we want to have a new saner/simpler `basic` schema?   Or `ecommerce` schema?   I also wonder if we should think about this in light of some of the work to have benchmarks....

Reply:
If I put on my client library developer hat, I'd say a "feature complete" configset with sample data that provides predictable results to test every possible use case. ;-)

I'm happy to join that discussion (if permitted) as an external stakeholder.